p256/src/arithmetic/projective.rs

```diff
@@ -284,7 +284,7 @@ impl ProjectivePoint {
     fn mul(&self, k: &Scalar) -> ProjectivePoint {
         let mut ret = ProjectivePoint::identity();
 
-        for limb in k.0.iter().rev() {
+        for limb in k.to_u64x4().iter().rev() {
             for i in (0..64).rev() {
```

Comment:
This would be one of the easiest things to switch to using 32-bit limbs on 32-bit targets, but I left it as close to the original implementation as possible for now.